### ISSUE TITLE
Help tests for Get-PSSessionCapability and New-PSRoleCapabilityFile fail as no description is found

### ISSUE THREAD

**Issue by adityapatwardhan** [PowerShell/PowerShell#2460]
Steps to reproduce
------------------
Execute tests from PowerShell\test\powershell\engine\HelpSystem.Tests.ps1

Expected behavior
-----------------
All tests pass

Actual behavior
---------------
2 tests fail

Environment data
----------------
```powershell
> $PSVersionTable
Name                           Value
----                           -----
SerializationVersion           1.1.0.1
BuildVersion                   3.0.0.0
PSRemotingProtocolVersion      2.3
WSManStackVersion              3.0
PSEdition                      Core
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
GitCommitId                    v6.0.0-alpha.10-119-g26c7321074f6ad662eada05444ae42578c7bf130
PSVersion                      6.0.0-alpha
CLRVersion
```

**Comment by iSazonov** [comment]
Add RequireAdminOnWindows tag would solve the problem.

**Comment by daxian-dbw** [comment]
Resolved by PR #2464
